### PR TITLE
Cards branch

### PR DESCRIPTION
**Pull request by Shivani-1524**
Things to keep in mind while reviewing :
1.  buttons dont have the hover effects as it hasnt been merged to the dev branch yet.
2. Flexible card is supposed to look like that as it is just a template that should be customised by the user.
3. the code snippets might look off but it will be fixed after I merge this branch.
4. will added my image responsive class after i merge branch.

### CODE REVIEW

File: Components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
+
+.money {
+  display: flex;
+  align-items: center;
+  font-size: 1.5rem;
+}
```
Comment (amantulsyan35) [inline Components/Card/card.css]: This naming for the card money is a bit vague, can be changed to something like card-money, ver-card-money or hor-card-money

File: documentation.html [hunk 2]
```diff
@@ -60,6 +60,81 @@
...
+                                            <div class="rating-row">
+                                                <i class="fas fa-star"></i>
+                                                <p class="sm-title">4.3</p>
+                                            </div>
+                                            <i class="fas fa-circle"></i>
+                                            <p class="review-num xsm-p grey-txt">(43)</p>
+                                        </div>
+                                        <div class="money mg-t-10">
+                                            <i class="fas fa-rupee-sign"></i>
+                                            <p class="rg-title">200.83</p>
```
Comment (amantulsyan35) [inline documentation.html]: The font-size of the rupee icon can be made smaller, if this is the original size intention

**Review by thenorthkun — APPROVED** [review]
Looks alright. You can merge this.

**Comment by prankurpandeyy** [comment]
looks great , optimize carbon code snippet so that your site loadig time can be reduced

File: Components/Card/card.css (hunk 1 above)
Comment (Shivani-1524) [reply Components/Card/card.css]: okay will do that

File: documentation.html (hunk 2 above)
Comment (Shivani-1524) [reply documentation.html]: im going to use Rs. instead

**Comment by Shivani-1524** [comment]
> looks great , optimize carbon code snippet so that your site loadig time can be reduced

i did add loading="lazy" if youknow of any other way please let me know

**Comment by Shivani-1524** [comment]
> 

okay! :)

**Pull request merged**